Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs

```diff
@@ -407,26 +417,35 @@ public bool GlobalUseThemeFormChromeBorderWidth
         /// Gets and sets the global flag that decides if palette colors are applied to toolstrips.
         /// </summary>
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Visible)]
-        public static bool ApplyToolstrips
+        public static bool ApplyToolStrips
```

Comment:
The rename of the Designer variable is a **Breaking Change** and should be undone !

Reply:
@Smurf-IV will fix on Tuesday